.github/workflows/builds_and_tests.yml

```diff
@@ -134,6 +134,22 @@ jobs:
           name: libwasmvm
           path: ./api/libwasmvm.x86_64.so
 
+  build_static_lib:
+    needs: build_shared_library
```

Comment:
I'm not sure, but, is it needed?

Reply:
Yes, I want to test that each PR builds correctly.

Reply:
Then, could we run `build_shared_library` and `build_static_lib` in parallel? As in my checking, it seems that `build_static_lib` doesn't have the dependency on `build_shared_library`.